### PR TITLE
Add Ukrainian Language

### PR DESCRIPTION
**Pull request by UKRMateGit**
Here is a full translation of the app in Ukrainian. If I forgot to add/modify anything else, please tell me.

### CODE REVIEW

**Comment by TheExentist151** [comment]
I don't think you need to translate README.md, because people can see it only in this repository, not in HMM itself.

**Review by thesupersonic16 — CHANGES_REQUESTED** [review]
At this point of time we do not translate readme's on HedgeModManager's repo, maybe we can in the future, but right now we don't, because of this we will require you to revert `README.md`'s changes before we can accept the PR. You are free however to make a fork or make a Ukrainian guide.

Also if possible having another user to check the translation would be preferred.

EDIT: I have also noticed one or more text not fitting in the buttons, please also double check all the text fits like the save and play button.

**Review by thesupersonic16 — APPROVED** [review]

**Comment by thesupersonic16** [comment]
Well that was a mistake oops. lol thought that was to approve that one review.

**Comment by thesupersonic16** [comment]
I will need you to also shorten the save and play text, then we can merge.

**Comment by UKRMateGit** [comment]
> I will need you to also shorten the save and play text, then we can merge.

I tried to short it, and done everything i can. Is that ok?

File: HedgeModManager/Languages/Languages.xaml [hunk 1]
```diff
@@ -6,6 +6,7 @@
     <local:LanguageList x:Key="Languages">
         <local:LangEntry FileName="en-AU" Name="English (Australia)"/>
         <local:LangEntry FileName="en-US" Name="English (United States)"/>
+        <local:LangEntry FileName="uk-UA" Name="Українська (Ukrainian)"/>
```
Comment (thesupersonic16) [inline HedgeModManager/Languages/Languages.xaml]: Just notice a minor issue here. Within the brackets should be the name of the region. In the case of uk-UA, it should be Ukraine as I have suggested below.

```suggestion
        <local:LangEntry FileName="uk-UA" Name="Українська (Ukraine)"/>
```

**Review by thesupersonic16 — CHANGES_REQUESTED** [review]
Noticed an issue with the name of the language. I have made a comment with a code suggestion.

**Comment by thesupersonic16** [comment]
> > I will need you to also shorten the save and play text, then we can merge.
> 
> I tried to short it, and done everything i can. Is that ok?

Try load the language on your end to see if it's fine. I do not understand Ukrainian.

**Review by thesupersonic16 — APPROVED** [review]

**Comment by UKRMateGit** [comment]
> > > I will need you to also shorten the save and play text, then we can merge.
> > 
> > 
> > I tried to short it, and done everything i can. Is that ok?
> 
> Try load the language on your end to see if it's fine. I do not understand Ukrainian.

Okay, but how do I add my translation to the app?  I downloaded the code from github, unpacked the archive, but I don't know how to run it.

**Comment by thesupersonic16** [comment]
> > > > I will need you to also shorten the save and play text, then we can merge.
> > > 
> > > 
> > > I tried to short it, and done everything i can. Is that ok?
> > 
> > 
> > Try load the language on your end to see if it's fine. I do not understand Ukrainian.
> 
> Okay, but how do I add my translation to the app? I downloaded the code from github, unpacked the archive, but I don't know how to run it.

Next to the exe create a folder called `Languages` and place your .xaml file in there.

**Comment by ChervZXC** [comment]
> > > > > I will need you to also shorten the save and play text, then we can merge.
> > > > 
> > > > 
> > > > I tried to short it, and done everything i can. Is that ok?
> > > 
> > > 
> > > Try load the language on your end to see if it's fine. I do not understand Ukrainian.
> > 
> > 
> > Okay, but how do I add my translation to the app? I downloaded the code from github, unpacked the archive, but I don't know how to run it.
> 
> Next to the exe create a folder called `Languages` and place your .xaml file in there.

Now there's error:
HMM Info:
```
    Version: 7.9-0
    Args: 
    StartDir: C:\Users\38096\Downloads\Kirby\
    Process Level: User
    Game: No Game Detected!
    Update Status: Up To Date
```

Exception:
```
    Type: LanguageLoadException
    Message: Failed to load language "Languages". Please make sure the file structure is correct.
    Source: HedgeModManager
    Function: System.Windows.ResourceDictionary GetLanguageResource(System.String)
    StackTrace: 
       at HedgeModManager.HedgeApp.GetLanguageResource(String culture) in E:\GitHub\HedgeModManager\HedgeModManager\HedgeApp.xaml.cs:line 418
       at HedgeModManager.HedgeApp.CountLanguages() in E:\GitHub\HedgeModManager\HedgeModManager\HedgeApp.xaml.cs:line 431
       at HedgeModManager.HedgeApp.Main(String[] args) in E:\GitHub\HedgeModManager\HedgeModManager\HedgeApp.xaml.cs:line 213
    InnerException: System.Windows.Markup.XamlParseException: 'Cannot create unknown type '{clr-namespace:HedgeModManager.Languages}LanguageList'.' Line number '6' and line position '6'. ---> System.Xaml.XamlObjectWriterException: 'Cannot create unknown type '{clr-namespace:HedgeModManager.Languages}LanguageList'.' Line number '6' and line position '6'.
   at System.Xaml.XamlObjectWriter.WriteStartObject(XamlType xamlType)
   at System.Windows.Markup.WpfXamlLoader.WriteStartObject(XamlReader xamlReader, XamlObjectWriter xamlWriter, XamlContextStack`1 stack)
   at System.Windows.Markup.WpfXamlLoader.TransformNodes(XamlReader xamlReader, XamlObjectWriter xamlWriter, Boolean onlyLoadOneNode, Boolean skipJournaledProperties, Boolean shouldPassLineNumberInfo, IXamlLineInfo xamlLineInfo, IXamlLineInfoConsumer xamlLineInfoConsumer, XamlContextStack`1 stack, IStyleConnector styleConnector)
   at System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)
   --- End of inner exception stack trace ---
   at System.Windows.Markup.XamlReader.RewrapException(Exception e, IXamlLineInfo lineInfo, Uri baseUri)
   at System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)
   at System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, Boolean skipJournaledProperties, Uri baseUri)
   at System.Windows.Markup.XamlReader.Load(XamlReader xamlReader, ParserContext parserContext)
   at System.Windows.Markup.XamlReader.Load(XmlReader reader, ParserContext parserContext, XamlParseMode parseMode, Boolean useRestrictiveXamlReader, List`1 safeTypes)
   at System.Windows.Markup.XamlReader.Load(Stream stream, ParserContext parserContext, Boolean useRestrictiveXamlReader)
   at HedgeModManager.HedgeApp.GetLanguageResource(String culture) in E:\GitHub\HedgeModManager\HedgeModManager\HedgeApp.xaml.cs:line 420
```

**Pull request merged**